src/main/java/com/example/napolinights/controller/MenuController.java

```diff
@@ -4,169 +4,156 @@
+     */
     public void initialize() {
-        // Load the categories and their items
+        // Set padding for the contentAnchorPane to provide spacing
+        contentAnchorPane.setPadding(new Insets(10, 50, 10, 50)); // Top, right, bottom, left padding
+
+        // Ensure the ScrollPane fits the width of its content
+        menuScrollPane.setFitToWidth(true);
+
+        // Load the menu items from the database
         loadMenuItems();
-        /*addCategory("Starters & Salads", Arrays.asList(
-                new MenuItem(Category.ENTREE.toString(), "Freshly cooked with mozzarella", 18.00, "path_to_image1"),
-                new MenuItem("Fried Green Olives", "Lightly crumbed and served with truffle mayo", 24.00, "path_to_image2")
-        ));
-
-        addCategory("WOODFIRED PIZZA", Arrays.asList(
-                new MenuItem("Margherita", "Traditional pizza with fresh tomato and mozzarella", 20.00, "path_to_image3"),
-                new MenuItem("Pepperoni", "Topped with spicy pepperoni and mozzarella", 22.00, "path_to_image4")
-        ));*/
+
+        // Ensure that the stage size is adjusted after the scene is loaded
+        Platform.runLater(() -> {
+            Stage stage = (Stage) menuScrollPane.getScene().getWindow();
+            stage.setMinWidth(800);
+            stage.setMinHeight(750);
+        });
     }
 
+    /**
+     * Loads the menu items from the database and groups them by category.
+     */
     private void loadMenuItems() {
+        // Get a connection to the database
         Connection connection = SqliteConnection.getInstance();
         MenuItemDAO menuItemDAO = new MenuItemDAO(connection);
+
+        // Fetch all menu items and group them by their category
         List<MenuItem> menuItems = menuItemDAO.fetchAllMenuItems();
         Map<Category, List<MenuItem>> groupedMenuItems = menuItems.stream()
                 .collect(Collectors.groupingBy(MenuItem::getCategory));
+
+        // Display the grouped menu items on the UI
         displayMenuItems(groupedMenuItems);
     }
 
+    /**
+     * Displays the menu items grouped by their categories. Each category is displayed
+     * with a label, and its items are displayed in a grid.
+     *
+     * @param groupedMenuItems A map of categories and their respective menu items.
+     */
     private void displayMenuItems(Map<Category, List<MenuItem>> groupedMenuItems) {
-        groupedMenuItems.forEach((category, items) -> {
-            // Create a Label for each category
-            Label categoryLabel = new Label(category.name());
-            categoryLabel.setStyle("-fx-font-size: 24px; -fx-font-weight: bold;");
+        // Define the desired order of the categories
+        List<Category> preferredCategoryOrder = List.of(
+                Category.ENTREE,
+                Category.PIZZA,
+                Category.PASTA,
+                Category.DESSERT,
+                Category.BEVERAGES
```

Comment:
Beverages doesn't exist in Category enum.  Either we need to extend Category or change this to either NON_ALCOHOLIC_DRINKS or 
COCKTAILS